pages/index.js

```diff
@@ -1,110 +1,67 @@
-              src="/assets/sports.jpg"
-              alt="Illustration of a green bicycle"
-            />
+            <h3 className="text-white text-center">
+              We build and buy technology to help city staff tackle <br></br>
+              Austin&apos;s mobility challenges.
+            </h3>
           </Col>
         </Row>
-        <Row className="py-3 text-primary bg-light">
-          <Col md={2}></Col>
-          <Col>
-            <h2>
-              <strong>We&apos;re here to help</strong>
-            </h2>
-          </Col>
+        <Row className="pt-5 pb-4 text-primary text-center">
+          <h2>
+            <strong>We&apos;re here to help</strong>
+          </h2>
         </Row>
-        <Row className="text-primary bg-light">
-          <Col md={2}></Col>
+        <Row xs={1} md={2} lg={4} className="mx-5 text-primary text-center">
           <ServiceItem
             content="Modernize software development processes while introducing good agile and human-centered design practices"
-            icon={<FaUsersCog style={{ verticalAlign: "top" }} />}
+            icon={
+              <IoIosPeople style={{ verticalAlign: "top", fontSize: "3rem" }} />
+            }
           />
-          <Col md={2}></Col>
           <ServiceItem
-            content="Digitize and streamline internal systems to save time and unlock insights from data"
-            icon={<FaBusinessTime style={{ verticalAlign: "top" }} />}
+            content="Improve public-facing services like websites or applications"
+            icon={<MdWeb style={{ verticalAlign: "top", fontSize: "3rem" }} />}
           />
-          <Col md={2}></Col>
-        </Row>
-        <Row className="pb-4 text-primary bg-light">
-          <Col md={2}></Col>
           <ServiceItem
-            content="Improve public-facing services like websites or applications"
-            icon={<MdWeb style={{ verticalAlign: "top" }} />}
+            content="Digitize and streamline internal systems to save time and unlock insights from data"
+            icon={<FaBriefcase style={{ verticalAlign: "top" }} />}
           />
-          <Col md={2}></Col>
           <ServiceItem
             content="Manage geospatial data and publish it on the web"
             icon={<FaGlobe style={{ verticalAlign: "top" }} />}
           />
-          <Col md={2}></Col>
-        </Row>
-        <Row className="pt-3 pb-4 text-primary">
-          <Col md={2}></Col>
-          <Col>
-            <h2>
-              <strong>Featured projects</strong>
-            </h2>
-          </Col>
-        </Row>
-        <Row>
-          {projects.map((project) => {
-            return project.isFeatured ? (
-              <Col key={project.number} md={3} className="m-0 p-2">
-                <IndexIsssueListItem type="project" issue={project} />
-              </Col>
-            ) : null;
-          })}
         </Row>
+        <hr className="mt-2 mx-4" />
+        <FeaturedSection />
```

Comment:
Calling this `FeaturedSection` now instead of `FeaturedCarousel` now that we don't have the carousel in mobile or tablet view